sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs

```diff
@@ -1542,6 +1548,51 @@ public virtual void RequestAuthorizationException(string identifier, string endp
                 WriteEvent(RequestAuthorizationExceptionEvent, identifier, endpoint, exception);
             }
         }
+
+        [Event(DrainLinkStartEvent, Level = EventLevel.Verbose, Message = "{0}: Starting drain operation.")]
+        public void DrainLinkStart(string identifier)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(DrainLinkStartEvent, identifier);
```

Comment:
```suggestion
                WriteEvent(DrainLinkStartEvent, identifier ?? string.Empty);
```

Reply:
We generally try to do the safety transforms at the lowest level, so that callers don't have to know how things are used and have less risk of unintentionally breaking things.

Leaving two comments to show the pattern, but we should apply to all `identifier` references.